src/app/public/modules/text-editor/defaults/font-size-list-defaults.ts

```diff
@@ -0,0 +1 @@
+export const FONT_SIZE_LIST_DEFAULTS: number[] = [6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 18, 20, 22, 24, 26, 28, 36, 48];
```

Comment:
Is this internal? If so - we need to add that JS Doc

Reply:
Good catch. Updated.